src/chocolatey/infrastructure.app/runners/ConsoleApplication.cs

```diff
@@ -85,9 +88,63 @@ public void run(string[] args, ChocolateyConfiguration config, Container contain
                                 }
                             }
                         },
-                        () => command.handle_validation(config),
+                        () =>
+                        {
+                            command.handle_validation(config);
+
+                            var validationResults = new List<ValidationResult>();
+                            var validations = container.GetAllInstances<IValidation>();
+
+                            foreach (var validation in validations)
+                            {
+                                validationResults.AddRange(validation.validate(config));
+                            }
+
+                            var validationErrors = report_validation_summary(validationResults);
+
+                            if (validationErrors != 0)
+                            {
+                                // NOTE: This is intentionally left blank

```

Comment:
Also mention why this is left blank ("as we reported the summary above")

Reply:
Was thinking on this after we set it that way and I think a reason would be helpful when we are in there later and see the note but then go to ask why.